ios/Classes/AblyFlutterPlugin.m

```diff
@@ -223,7 +223,7 @@ -(instancetype)initWithChannel:(FlutterMethodChannel *const)channel streamsChann
     if (!self) {
         return nil;
     }
-    _ably = [AblyFlutter sharedInstance];
+    _ably = [[AblyFlutter sharedInstance] setChannel: channel];
```

Comment:
As commented on earlier where you define `setChannel:`...
At a minimum I would expect something like:

```suggestion
    _ably = [AblyFlutter sharedInstance];
    [_ably setChannel:channel];
```

Reply:
772c54ec3cada003fed9a2b8230cfd3f5c51822f